config_src/drivers/FMS_cap/ocean_model_MOM.F90

```diff
@@ -504,9 +512,17 @@ subroutine update_ocean_model(Ice_ocean_boundary, OS, Ocean_sfc, time_start_upda
   ! Translate Ice_ocean_boundary into fluxes and forces.
   call get_domain_extent(Ocean_sfc%Domain, index_bnds(1), index_bnds(2), index_bnds(3), index_bnds(4))
 
+  if (OS%use_homogenized_IOB) &
+    call homogenize_IOB(Ice_ocean_boundary, Ice_ocean_boundary_homogenized, OS%grid, index_bnds)
```

Comment:
@breichl , what is the reason for doing this on an extra copy of IOB? I think if we homogenize the fields on the origin Ice_ocean_boundary (rather than the new Ice_ocean_boundary_homogenized ) everything would work fine, including the (generic) tracer fluxes. The problem with  Ice_ocean_boundary_homogenized is that it does not have the hidden member "fluxes" allocated.

Reply:
One issue with modifying IOB itself is that it is intent(in) in update_ocean_model, I had to make it intent(inout). But after that all seem to have worked fine, including the homogenization and the generic tracers.

Reply:
Okay, thanks Niki.  I made the copy to avoid modifying this structure since it is intent(in).  If we are comfortable making it intent(inout) then this solution works for me.  Do the generic tracers fields also need to be homogenized?  I'm not sure how difficult that would be or if it is necessary here.